### PR TITLE
Update memory-integrity.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4042
Added note.

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix and @JohanFreelancer9 for copy review please. Than you.

File: windows/security/threat-protection/windows-defender-exploit-guard/memory-integrity.md [hunk 1]
```diff
@@ -24,5 +24,6 @@ manager: dansimp
 
 Memory integrity is a powerful system mitigation that leverages hardware virtualization and the Windows Hyper-V hypervisor to protect Windows kernel-mode processes against the injection and execution of malicious or unverified code. Code integrity validation is performed in a secure environment that is resistant to attack from malicious software, and page permissions for kernel mode are set and maintained by the Hyper-V hypervisor. Memory integrity helps block many types of malware from running on computers that run Windows 10 and Windows Server 2016.
 
-
+> [!NOTE]
+> For more information see [Device protection in Windows Defender Security Center](https://support.microsoft.com/help/4096339/windows-10-device-protection-in-windows-defender-security-center).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-exploit-guard/memory-integrity.md]:
```suggestion
> For more information, see [Device protection in Windows Defender Security Center](https://support.microsoft.com/help/4096339/windows-10-device-protection-in-windows-defender-security-center).
```

**Comment by jvsam** [comment]
Hi @levinec this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [commits](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4359/commits) and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4359/files).

**Comment by jvsam** [comment]
Hi again @levinec, kindly review this PR when time permits. Thank you.

**Comment by jvsam** [comment]
Thank you @levinec for merging this PR. Really appreciate it!

**Pull request merged**